CreateService/handler.ts

```diff
@@ -144,22 +174,30 @@ export function CreateServiceHandler(
   generateObjectId: ObjectIdGenerator,
   productName: NonEmptyString
 ): ICreateServiceHandler {
-  return (_, __, ___, userAttributes, servicePayload) => {
+  return (context, __, ___, userAttributes, servicePayload) => {
     const subscriptionId = generateObjectId();
+    context.log.info(
+      `${logPrefix}| Creating new service with subscriptionId=${subscriptionId}`
+    );
     return createSubscriptionTask(
+      context,
       apiClient,
       userAttributes.email,
       subscriptionId,
       productName
     )
       .chain(subscription =>
-        createServiceTask(apiClient, servicePayload, subscriptionId).map(
-          service =>
-            ResponseSuccessJson({
-              ...service,
-              primary_key: subscription.primary_key,
-              secondary_key: subscription.secondary_key
-            })
+        createServiceTask(
+          context,
```

Comment:
passing `context.log` instead of the whole `context` would be better. 
moreover if it's always the same pattern you can have a log method bound to context ie:

```ts
const log = (context, name) => (errs) => `${logPrefix}|${name}|ERROR=${JSON.stringify(errs)`
```

then pass `log(context, "CreateSubscription")` as argument and call `fold(log, ...)`

Reply:
Fixed in [d7f70c2](https://github.com/pagopa/io-functions-services/pull/68/commits/d7f70c2bb120fe7b97cf5a1bec35ce16ad528640)